app/src/main/java/beepbeep/pixels/feed/FeedActivity.kt

```diff
@@ -0,0 +1,66 @@
+package beepbeep.pixels.feed
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.view.View
+import beepbeep.pixels.R
+import beepbeep.pixels.shared.extension.addTo
+import io.reactivex.Observable
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.subjects.BehaviorSubject
+import kotlinx.android.synthetic.main.content_home.loadMoreButton
+import kotlinx.android.synthetic.main.content_home.loadingProgress
+import kotlinx.android.synthetic.main.content_home.refreshButton
+import kotlinx.android.synthetic.main.content_home.resultText
+
+class FeedActivity : AppCompatActivity(), FeedContract.Input {
+
+    private val interactor: FeedContract.Output by lazy {
+        FeedInteractor(this, FakeFeedRepository())
+    }
+
+    private val refreshable = BehaviorSubject.create<View>()
```

Comment:
This will be cleaned up in the next PR!

Reply:
Aha subject alert! You make me start to think this is bad now... but my RxKungFu is not too good to avoid it always yet..